src/api/spec/jobs/status_history_rescaler_job_spec.rb

```diff
@@ -3,32 +3,52 @@
 RSpec.describe StatusHistoryRescalerJob, type: :job do
   include ActiveJob::TestHelper
 
+  let(:status_histories) { StatusHistory.where(key: 'busy_x86_64').order(:time) }
+
   describe '#rescale' do
-    before do
-      Timecop.freeze(2010, 7, 12)
-
-      now = Time.now.to_i - 2.days
-      StatusHistory.transaction do
-        1000.times do |i|
-          StatusHistory.create(time: now + i, key: 'idle_x86_64', value: i)
-        end
+    context 'newer than 2 days events' do
+      before do
+        5.times { |i| StatusHistory.create(time: (Time.now.utc - i.seconds).to_i, key: 'busy_x86_64', value: 10 * i) }
```

Comment:
Those records are kept intact. So no reduction is expected. Therefore the number of records doesn't change